pyfr/plugins/integrate.py

```diff
@@ -54,17 +59,43 @@ def __init__(self, intg, cfgsect, suffix=None):
 
         # Prepare the per element-type info list
         self.eleinfo = []
+        self.m0s = []
         for (ename, eles), (eset, emask) in zip(system.ele_map.items(), rinfo):
-            # Locations of each solution point
-            ploc = eles.ploc_at_np('upts')[..., eset]
-            ploc = ploc.swapaxes(0, 1)
+            # Obtain quadrature info
+            rname = self.cfg.get(f'solver-elements-{ename}', 'soln-pts')
 
-            # Jacobian determinants
-            rcpdjacs = eles.rcpdjac_at_np('upts')[:, eset]
+            if self.qdeg:
```

Comment:
Does this branch handle element subsetting correctly?

Reply:
No it didn't. Fixed it with the new commit.